src/containers/shared/components/Transaction/CredentialAccept/Simple.tsx

```diff
@@ -0,0 +1,35 @@
+import { useTranslation } from 'react-i18next'
+import { type CredentialAccept } from 'xrpl'
+import { TransactionSimpleComponent, TransactionSimpleProps } from '../types'
+import { SimpleRow } from '../SimpleRow'
+import { convertHexToString } from '../../../../../rippled/lib/utils'
+
+const Simple: TransactionSimpleComponent = (
+  props: TransactionSimpleProps<CredentialAccept>,
+) => {
+  const { t } = useTranslation()
+  const { data } = props
+  const { Account, Issuer, CredentialType } = data.instructions
```

Comment:
nit: I recommend you do something like this so that the variables aren't confused with classes
```suggestion
  const { Account: account, Issuer: issuer, CredentialType: credentialType } = data.instructions
```

Reply:
good idea, have added this in the new txs